src/main/java/domain/AttendanceStatus.java

```diff
@@ -0,0 +1,40 @@
+package domain;
+
+import java.time.LocalDateTime;
+import java.util.Arrays;
+import java.util.List;
+
+public enum AttendanceStatus {
+    ATTENDANCE("출석", 0),
+    LATE("지각", 5),
+    ABSENCE("결석", 30);
+
+    String name;
+    int boundary;
```

Comment:
접근제한자를 default로 두신 이유가 있을까요? 
또한 enum class의 상태 값들은 상태가 한번 지정되면 변경되지 않을테니 `final`로 선언해주면 좋을 것 같습니다. 

```suggestion
    private final String name;
    private final int boundary;
```

Reply:
> 접근제한자를 default로 두신 이유가 있을까요? 

급하다 보니, 빠뜨린 것 같습니다. 

> enum class의 상태 값들은 상태가 한번 지정되면 변경되지 않을테니 final로 선언해주면 좋을 것 같습니다.

enum 내부 필드는 상수 처리 되어있는 줄 알았는데, 말씀해주셔서 테스트 해보니 변경이 되더라구요. 리뷰 감사합니다 !!